myhoard/basebackup_operation.py

```diff
@@ -315,16 +323,25 @@ def _process_output_line_file_finished(self, line):
         self.current_file = None
         return True
 
-    def _process_output_line_binlog_info(self, line):
-        match = self.binlog_info_re.search(line)
-        if match:
-            self.binlog_info = {
-                "file_name": match.group("fn"),
-                "file_position": int(match.group("pos")),
-                "gtid": match.group("gtid"),
-            }
-            self.log.info("binlog info: %r", self.binlog_info)
-        return match
+    def _process_binlog_info(self):
+        if not self.lsn_dir:
+            return
+
+        binlog_file_path = os.path.join(self.lsn_dir, "xtrabackup_info")
+        try:
+            with open(binlog_file_path, "r") as binlog_file:
+                for line in binlog_file.readlines():
+                    match = self.binlog_info_re.search(line)
+                    if match:
+                        self.binlog_info = {
+                            "file_name": match.group("fn"),
+                            "file_position": int(match.group("pos")),
+                            "gtid": match.group("gtid"),
+                        }
+                        self.log.info("binlog info: %r", self.binlog_info)
+                        break
+        except FileNotFoundError:
+            pass
```

Comment:
This error should probably not be ignored as we'll end up missing critical info if it fails.